.github/workflows/label-flaky-test-issue.yml

```diff
@@ -0,0 +1,49 @@
+name: Label Flaky Test Issues
+
+on:
+  issues:
+    types: [opened, labeled]
+
+jobs:
+  label:
+    if: github.event.label.name == 'flaky-test'
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Extract labels
+        id: extract-labels
+        env:
+          BODY: ${{ github.event.issue.body }}
+        run: |
+          BODY="${BODY//$'\n'/'\n'}"
+
+          declare -A platform2label
+
+          platform2label["AIX"]="aix";
+          platform2label["FreeBSD"]="freebsd";
+          platform2label["Linux ARM64"]="linux";
+          platform2label["Linux ARMv7"]="arm";
+          platform2label["Linux PPC64LE"]="ppc";
+          platform2label["Linux s390x"]="s390";
+          platform2label["Linux x64"]="linux";
+          platform2label["macOS ARM64"]="macos";
+          platform2label["macOS x64"]="macos";
+          platform2label["SmartOS"]="smartos";
+          platform2label["Windows"]="windows";
+
+          # sed is cleaning up the edges
+          PLATFORMS=$(echo $BODY | sed 's/^.*Platform\\n\\n//' | sed 's/\(, Other\)\?\\n\\n.*$//') 2> /dev/null
+          readarray -d , -t list <<< "$PLATFORMS"
+          labels=
+          for row in "${list[@]}"; do \
+            platform=$(echo $row | xargs); \
+            labels="${labels},${platform2label[$platform]}"; \
+          done;
+
+          echo "::set-output name=LABELS::${labels}"
+
+      - name: Add labels
+        uses: andymckay/labeler@master
+        with:
+          add-labels: ${{ steps.extract-labels.outputs.LABELS }}
```

Comment:
Can we use `gh` CLI instead please?

Reply:
You meant: `gh issue edit`? I'm afraid it won't work in an easy way.

Reply:
What do you mean? Adding an external untrusted action is a blocker, that won't work either.

Reply:
If `gh issue edit --add-label` won't work, perhaps https://github.com/nodejs/node-pr-labeler can be modified (or forked as node-issue-labeler) so we're using an action we own? Although I imagine `gh issue edit --add-label` should work. See .github/workflows/comment-labeled.yml for an example of `gh issue comment` which hopefully is pretty close to what we need?

Reply:
There is also this action: https://github.com/github/issue-labeler which can label an issue according to its content with regex as this workflow does. But unfortunately you'll have to duplicate a bit for each OS.

Reply:
I've updated the code to use `gh issue edit --add-label`. It's working fine (check the lab repo)